terraform/variables.tf

```diff
@@ -0,0 +1,40 @@
+variable "profile" {
+  default = "default"
+}
+
+variable "region" {
+  default = "us-east-1"
+}
+
+variable "aws_ecs_ami" {
+  # ECS-optimized AMI for us-east-1.
+  default = "ami-b2df2ca4"
+}
+
+variable "ecs_instance_profile" {
+  default = "arn:aws:iam::896538046175:instance-profile/terraform-wzxkyowirnachcosiqxrriheki"
+}
+
+variable "ecs_service_role" {
+  default = "arn:aws:iam::896538046175:role/ecs_service_role"
+}
+
+variable "subnet_id" {
```

Comment:
Considering the site and demos are the only things GT deploys long-term, I think it makes more sense to tweak the GT AWS account as needed.